Yosemite/YosemiteTests/Stores/ProductVariationStoreTests.swift

```diff
@@ -290,6 +290,153 @@ final class ProductVariationStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    // MARK: - ProductVariationAction.retrieveProductVariation
+
+    /// Verifies that `ProductVariationAction.retrieveProductVariation` effectively persists the retrieved product variation.
+    ///
+    func test_retrieveProductVariation_effectively_persisted() throws {
+        // Given
+        let store = ProductVariationStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
+        let sampleProductVariationID: Int64 = 1275
+
+        network.simulateResponse(requestUrlSuffix: "products/\(sampleProductID)/variations/\(sampleProductVariationID)", filename: "product-variation")
```

Comment:
nit: it'd be nice to use `MockProductVariationsRemote` instead of the network response so that we can return any kind of result (same for the other instances in this test class)